maturity-level-two/README.md

```diff
@@ -11,3 +11,42 @@ You should:
 - Add your OpenAPI specs to source control given this is part of your application
 - Validate changes to your OpenAPI specs to avoid specification violations ([user guide](docs/validating-open-api-specs.md))
 - Unit test Open API validation to automatically detect breaking changes
+
+ ## Content negotiation
+ With content negotiation we detremine in which format we'd like to receive requests and in which format we'd like to return our responses. For REST API's common request formats are JSON and XML, the most common response formats are JSON, XML and JSON. When someone does an http request to your api he should use the 'Content-Type' header to specify in which format the request is done and use the 'Accept' header to determine in which format he expects a response. Depending on these headers your api will decide in which format the request is send in in which format it will return it's response. When a format is not supported you should return the 406 error response code (not acceptable). When no 'Accept' header is defined a default response format should be used. Becaus of it's lightness and fastness, JSON is getting the standard the last years. But in certain situations other formats still have their advantages.
```

Comment:
I've improved the readability a bit, what do you think?

```suggestion
With content negotiation we determine in what format we'd like to receive requests  & responses. For REST API's, the most common formats are JSON and XML.

As an API consumer, you can specify what format you are expecting by adding HTTP headers:

- `Content-Type` - Specify the format of the request
- `Accept` - Specify the requested format of the response. Default format will be used when not specified.

When a format is not supported, the API should return an [HTTP 406 Not Acceptable](https://httpstatuses.com/406).

Because of it's lightness and fastness, JSON has become the standard over the last couple of years but in certain situations other formats still have their advantages.
```